spring-kafka/src/main/java/org/springframework/kafka/listener/AcknowledgingMessageListener.java

```diff
@@ -30,13 +28,6 @@
  * @author Marius Bogoevici
  * @author Gary Russell
  */
-public interface AcknowledgingMessageListener<K, V> {
-
-	/**
-	 * Executes when a Kafka message is received.
-	 * @param record the Kafka message to be processed
-	 * @param acknowledgment a handle for acknowledging the message processing
-	 */
-	void onMessage(ConsumerRecord<K, V> record, Acknowledgment acknowledgment);
+public interface AcknowledgingMessageListener<K, V> extends IAcknowledgingMessageListener<ConsumerRecord<K, V>> {
```

Comment:
I understand that it is late already, but, eh..., it isn't `MessageListener`. `ConsumerRecord` - right, but not for message...